app/install

```diff
@@ -30,7 +30,11 @@ if [ "$(uname)" == "Darwin" ]; then
 fi
 
 ln -snf "$APP_DIR/cli" "$FOLDER/$NAME"
-
+if [[ $? == 0 ]]; then
+    echo "Succesfully installed repos in your folder $FOLDER"
```

Comment:
use the logging capabilities, by using `info "..."`

Reply:
Done in commit 5a74f38321541718cfe901291b5972d728f3d7fc